### ISSUE TITLE
All unhandled exceptions ignored

### ISSUE THREAD

**Issue by terabitti** [GDay/django-q2#48]
Do I miss something or is Django Q(2) completely ignoring any **unhandled** exceptions occurred in running tasks? Is there a simple way to log all of those errors (without using Sentry or Rollbar)?

It only shows errors like "Could not process 'task'. Check the location of the function and the args/kwargs." which seems to be false and misleading.

For a workaround, you have to manually catch all the exceptions in every task:

```python
def task():
    try:
        do_something_which_may_fail()
    except Exception:
        logger.exception("Something happened")
```

**Comment by GDay** [comment]
Thanks. You are indeed correct. I will get this fixed ASAP.

**Comment by jochemfranken** [comment]
@GDay Thanks for fixing this so quickly!

**Comment by GDay** [comment]
@jochemfranken @terabitti This fix has been released under v1.4.7! Sorry for the slight delay on this!

**Comment by GDay** [comment]
@Stephane-Ag there is no option for that at the moment. You would indeed need to add a wrapper function for that. 
Catching errors is not really flexible at the moment: 
https://github.com/django-q2/django-q2/blob/d8cc0cc4f4421390be8fe05325a4f2af8ddcaa2c/django_q/worker.py#L84-L95
